### PR TITLE
RAINCATCH-1274

### PR DESCRIPTION
**Pull request by austincunningham**
## Motivation:
need instructions for using wfm on RHMAP

## What
instructions on using wfm templates and steps to setup the applications

- [x] Documentation

### CODE REVIEW

File: docs/workforce-management-framework/rhmap/using-wfm-templates-on-rhmap.adoc [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+== Setting up the WFM Portal App
+
+- 1. Select the `WFM Portal App` 
+- 2. To deploy the Portal appliction select `Deploy` on the left navigation panel
+- 3. On the Deploy screen in the `Choose a Runtime` dropdown select `Node.js - 6.1.9`
+- 4. Click on the `Deploy Portal App`
+- 5· Once deployed return to the Project screen by clicking on `Apps, Cloud Apps & Services`
+
+NOTE: New changes to the WFM Portal App code base to be deployed
```
Comment (wtrocki) [inline docs/workforce-management-framework/rhmap/using-wfm-templates-on-rhmap.adoc]: What you mean by this: `New changes to the WFM Portal App code base to be deployed`

File: docs/workforce-management-framework/rhmap/using-wfm-templates-on-rhmap.adoc [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- 1. In the platform select `Projects` in the top navigation bar
+- 2. In the Projects screen select `New Project`
+- 3. In the `Choose a Template` screen select the `WFM Project` by clicking `Choose`
+- 4. Name your Project and click on the `Create`
+
+NOTE: You can apply different names to the individual appication within the Project
+
+- 5. Once the Project is created click `Finish` and the Project will render with the three application
+
+== Setting up the WFM Cloud App
```
Comment (wtrocki) [inline docs/workforce-management-framework/rhmap/using-wfm-templates-on-rhmap.adoc]: WFM server :P

File: docs/workforce-management-framework/rhmap/using-wfm-templates-on-rhmap.adoc [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- 3. On the Deploy screen in the `Choose a Runtime` dropdown select `Node.js - 6.1.9`
+- 4. Click on the `Deploy Portal App`
+- 5· Once deployed return to the Project screen by clicking on `Apps, Cloud Apps & Services`
+
+NOTE: New changes to the WFM Portal App code base to be deployed
+
+== Setting up the WFM Mobile App
+
+- 1. Select the `WFM Mobile App`
+- 2. Click on `Build` and use the RHMAP link:https://access.redhat.com/documentation/en-us/red_hat_mobile_application_platform/4.5/html-single/getting_started/#run-the-client-app-on-a-mobile-device[guide] for building a mobile binary on the platform.
```
Comment (wtrocki) [inline docs/workforce-management-framework/rhmap/using-wfm-templates-on-rhmap.adoc:43]: DO not like form here but I will leave that for John to fix it.

File: docs/workforce-management-framework/rhmap/using-wfm-templates-on-rhmap.adoc [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+Alternively you can build your binarys from  the link:https://access.redhat.com/documentation/en-us/red_hat_mobile_application_platform/4.5/html-single/local_development_guide/#building-an-app-binary[CLI]
+
+NOTE: New changes to the WFM Mobile App code base need to be applied to the `www` directory updated with the `grunt wfmTemplate:build` command
```
Comment (wtrocki) [inline docs/workforce-management-framework/rhmap/using-wfm-templates-on-rhmap.adoc]: That's not true. Grunt browserify. So you need to run grunt default task. wfm template build will update only templates.

File: docs/workforce-management-framework/rhmap/using-wfm-templates-on-rhmap.adoc (hunk 1 above)
Comment (wtrocki) [reply docs/workforce-management-framework/rhmap/using-wfm-templates-on-rhmap.adoc]: Let's use: "Everytime you change content of the app your application needs to be deployed"

File: docs/workforce-management-framework/rhmap/using-wfm-templates-on-rhmap.adoc (hunk 4 above)
Comment (wtrocki) [reply docs/workforce-management-framework/rhmap/using-wfm-templates-on-rhmap.adoc]: Execute `grunt`

**Comment by wtrocki** [comment]
@austincunningham Very little changes required and most of them are just minor fixes.  Added tech-approved label, as after fixing those everything will be ok.

**Comment by wtrocki** [comment]
Good to go!

File: docs/workforce-management-framework/rhmap/using-wfm-templates-on-rhmap.adoc [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+- 4. Click on the `Deploy Portal App`
+- 5· Once deployed return to the Project screen by clicking on `Apps, Cloud Apps & Services`
+
+NOTE: Everytime you change content of the app your application needs to be deployed
+
+== Setting up the WFM Mobile App
+
+- 1. Select the `WFM Mobile App`
+- 2. Click on `Build` and use the RHMAP link:https://access.redhat.com/documentation/en-us/red_hat_mobile_application_platform/4.5/html-single/getting_started/#run-the-client-app-on-a-mobile-device[guide] for building a mobile binary on the platform.
+Alternively you can build your binarys from  the link:https://access.redhat.com/documentation/en-us/red_hat_mobile_application_platform/4.5/html-single/local_development_guide/#building-an-app-binary[CLI]
```
Comment (witmicko) [inline docs/workforce-management-framework/rhmap/using-wfm-templates-on-rhmap.adoc:44]: binaries

**Pull request closed without merging**